frontend/lib/src/components/input/input_field.dart

```diff
@@ -84,6 +85,11 @@ class InputField extends StatelessWidget {
                 ),
               ),
               obscureText: obscureText ?? false,
+              inputFormatters: <TextInputFormatter>[
```

Comment:
use reusable component here, InputField

Reply:
inputFormatters use for restriction example in the TextInputType.number, it uses a FilteringTextInputFormatter.digitsOnly formatter to allow only digits as input.